requirements.txt

```diff
@@ -145,3 +149,82 @@ google-auth-httplib2==0.0.3 \
 psycopg2-binary==2.8.3 \
  --hash=sha256:d305313c5a9695f40c46294d4315ed3a07c7d2b55e48a9010dad7db7a66c8b7f\
  --hash=sha256:19d013e7b0817087517a4b3cab39c084d78898369e5c46258aab7be4f233d6a1
+
+# required for wfdb
+wfdb==3.2.0 \
+ --hash=sha256:0c28b2a74e4cb79209272d570b694f8509c54c9d4bea668804705576eb696823
+cycler==0.10.0 \
```

Comment:
There is no need to include packages that are dependencies of WFDB because they will be installed as needed. Removing the dependencies may help with the installation errors that you were getting.